scripts/validate-autoinstall-user-data.py

```diff
@@ -27,64 +27,224 @@
 """
 
 import argparse
+import asyncio
 import io
-import json
+import sys
+import tempfile
+import traceback
+from argparse import Namespace
+from pathlib import Path
+from textwrap import dedent
+from typing import Any
 
-import jsonschema
 import yaml
 
+# Python path trickery so we can import subiquity code and still call this
+# script without using the makefile
+scripts_dir = sys.path[0]
+subiquity_root = Path(scripts_dir) / ".."
+curtin_root = subiquity_root / "curtin"
+probert_root = subiquity_root / "probert"
+# At the very least, local curtin needs to be in the front of the python path
+sys.path.insert(0, str(subiquity_root))
+sys.path.insert(1, str(curtin_root))
+sys.path.insert(2, str(probert_root))
```

Comment:
I don't love this but I don't have any better suggestions, so :laughing:

Reply:
I agree, it's not great. I can't yet think of any less worse alternatives, except for packaging it into the snap somehow.